ansible/roles/ooni-measurements/templates/ngx-oomsm-web

```diff
@@ -49,6 +49,11 @@ server {
 
     {{ c.location_letsencrypt() }}
 
+    location = /robots.txt {
+       add_header Content-Type text/plain;
+       return 200 "User-agent: *\nDisallow: /\nAllow: /docs";
```

Comment:
The API docs are in: https://api.ooni.io/api/

Also we should allow scraping the index of the API.

Reply:
I would say we should make this be like follows:

```
User-agent: *
Allow: /
Disallow: /api/v1/
Disallow: /api/_/
Disallow: /files/
Disallow: /stats/
```